dev/NavigationView/NavigationView_rs2_themeresources.xaml

```diff
@@ -95,7 +95,7 @@
             <StaticResource x:Key="NavigationViewItemBackgroundSelectedPressed" ResourceKey="SystemControlHighlightListMediumRevealBackgroundBrush" />
             <StaticResource x:Key="NavigationViewItemBackgroundSelectedDisabled" ResourceKey="SystemControlTransparentRevealBackgroundBrush" />
 
-            <StaticResource x:Key="NavigationViewItemBorderBrush" ResourceKey="SystemControlBackgroundTransparentRevealBorderBrush" />
+            <StaticResource x:Key="NavigationViewItemBorderBrush" ResourceKey="SystemControlTransparentBrush" />
```

Comment:
@MikeHillberg would you recommend that this resource for themes other than HC also be changed to SystemControlTransparentBrush, to maintain consistency?

Reply:
@kikisaints, same question to you about whether to update all three resource mappings? Or should we update only the HC one?

Reply:
@YuliKl I don't think it's necessary to adjust all theme brushes if we just change HC ones. I know there are a lot of places where we just adjust HC and not the others.

In additiona, wouldn't changing the others to SystemControlTransparentBrush also lose the Reveal effect in dark and light theme?